ApiReview/API.Polly.Core/NoDocs/Polly.Hedging/OnHedgingArguments.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+using Polly.Utils;
+
+namespace Polly.Hedging;
+
+public readonly struct OnHedgingArguments<TResult>
+{
+    public Outcome<TResult> Outcome { get; }
+    public ResilienceContext Context { get; }
+    public int AttemptNumber { get; }
+    public bool HasOutcome { get; }
```

Comment:
This tells the consumer that no outcome was produced in previous attempt due to operation taking too long. 

Maybe find a better name or just make `Outcome` property nullable (since now we have it).